### PR TITLE
Mes 9916 | Biome no explicit any

### PR DESCRIPTION
**Pull request by gittins7**
## Description
Implemented Biome's no explicit any rule
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/test-report/cat-a-mod1/components/speed-check/speed-check.ts
Comment (RLCorp) [inline src/app/pages/test-report/cat-a-mod1/components/speed-check/speed-check.ts:1]: Are there not predefined types we can use for the functions in this file?

File: src/app/pages/view-test-result/view-test-result.page.ts [hunk 1]
```diff
@@ -87,7 +87,8 @@ export class ViewTestResultPage extends BasePageComponent implements OnInit {
     this.subscription = this.searchProvider
       .getTestResult(this.applicationReference, this.authenticationProvider.getEmployeeId())
       .pipe(
-        map((response: HttpResponse<any>): string => response.body),
+        map((response: HttpResponse<string>): string => response.body),
+        tap((data) => console.log(data)),
```
Comment (RLCorp) [inline src/app/pages/view-test-result/view-test-result.page.ts]: Remove this console log please

File: src/app/pages/waiting-room-to-car/components/vehicle-registration/vehicle-registration.ts
Comment (RLCorp) [inline src/app/pages/waiting-room-to-car/components/vehicle-registration/vehicle-registration.ts:1]: Re-implement clearData and abortMOTCall please, they are needed for MOT to function properly

File: src/app/providers/device-authentication/__tests__/device-authentication.spec.ts
Comment (RLCorp) [inline src/app/providers/device-authentication/__tests__/device-authentication.spec.ts:1]: Remove comments please

File: src/app/providers/search/__mocks__/search.mock.ts
Comment (RLCorp) [inline src/app/providers/search/__mocks__/search.mock.ts:1]: Is these not just an observable<HttpResponse>?

File: src/components/common/chart/chart.ts [hunk 2]
```diff
@@ -4,6 +4,12 @@ import ApexCharts from 'apexcharts';
 import { isEqual } from 'lodash-es';
 import { ApexAxisChartSeries, ApexNonAxisChartSeries, ApexOptions, ChartType } from 'ng-apexcharts';
 
+interface ExaminerRecord {
```
Comment (RLCorp) [inline src/components/common/chart/chart.ts]: This new type seems to be unused?

File: src/components/common/signature-area/__tests__/signature-area.spec.ts
Comment (RLCorp) [inline src/components/common/signature-area/__tests__/signature-area.spec.ts:1]: Is there no built in type for this? (Could be NgSignaturePadOptions)?

**Review by RLCorp — APPROVED** [review]

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**